### ISSUE TITLE
TrailingSemicolon false positive for multiline variable declarations

### ISSUE THREAD

**Issue by adidahiya** [brigade/scss-lint#627]
This code reports a TrailingSemicolon failure:

```sass
$foo: (
  "a": (
    "b",
    "c"
  )
);
```

while this does not:

```sass
$foo: (
  "a": (
    "b",
    "c"
  ));
```

I expect the first syntax to be valid.

scss-lint v0.42.2
